api/http/dapps.js

```diff
@@ -24,12 +21,9 @@ function DappsHttpApi (dappsModule, app) {
 
 	var router = new Router();
 
-	router.map(dappsModule.internal, {
-		'get /categories': 'categories',
-	});
-
-	router.get('/', httpApi.middleware.sanitize('query', schema.list, dappsModule.internal.list));
-	router.get('/get', httpApi.middleware.sanitize('query', schema.get, dappsModule.internal.get));
+	router.map(dappsModule.shared, {
+		'get /': 'list'
```

Comment:
I suggest to follow same pattern than other modules: `getDapps` instead of `list`